generators/php/codegen/src/context/AbstractPhpGeneratorContext.ts

```diff
@@ -94,6 +130,69 @@ export abstract class AbstractPhpGeneratorContext<
         }
     }
 
+    public isEnum(typeReference: TypeReference): boolean {
```

Comment:
fine for now, but would like to revisit these; i think i have ideas for a bit more code reuse